versions.props

```diff
@@ -11,26 +11,26 @@ com.google.auto:auto-common = 0.10
 com.google.auto.service:auto-service = 1.0-rc6
 com.google.code.findbugs:findbugs-annotations = 3.0.1
 com.google.code.findbugs:jsr305 = 3.0.2
-com.google.dagger:dagger = 2.19
+com.google.dagger:dagger = 2.25.3
 com.google.dagger:dagger-compiler = 2.17
-com.google.errorprone:error_prone_annotations = 2.3.3
-com.google.errorprone:error_prone_core = 2.3.3
+com.google.errorprone:error_prone_annotations = 2.3.4
+com.google.errorprone:error_prone_core = 2.3.4
```

Comment:
Do we want to keep this at 2.3.3 per https://github.com/palantir/gradle-baseline/issues/1113